components/board/server.go

```diff
@@ -207,6 +207,34 @@ func (s *serviceServer) GetDigitalInterruptValue(
 	return &pb.GetDigitalInterruptValueResponse{Value: val}, nil
 }
 
+func (s *serviceServer) StreamTicks(
+	req *pb.StreamTicksRequest,
+	server pb.BoardService_StreamTicksServer,
+) error {
+	b, err := s.coll.Resource(req.Name)
+	if err != nil {
+		return err
+	}
+
+	ticksChan := make(chan Tick, 1024)
```

Comment:
Similar question here: why does `ticksChan` have a capacity for 1024 Ticks? I'd kinda expect a capacity of 0 should work correctly.